applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle DESKTOP_DVFS_STATE_$(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "timeout"
```

Comment:
Some states do not use the timeout at all (e.g. USB). We should avoid defining this Kconfig symbol if timeout for a given DVFS state is not supported.